classes/class4/homework-soln/public/javascripts/handlers.js

```diff
@@ -0,0 +1,69 @@
+var HANDLERS = {
+	click: {
+	// Define all click handlers here
+		orderOpt: function(event) {
```

Comment:
Fun fact for the day: spaces and tabs look the same in Sublime, but Github knows better...